### PR TITLE
add recent updates to perf benchmark

### PR DESCRIPTION
**Pull request by richardwxn**
1. update installation profiles corresponding to new operator API
2. fix the git sha issue: https://github.com/istio/istio/issues/20185
3. update telemetryv2 installation

### CODE REVIEW

File: perf/istio-install/setup_istio.sh [hunk 1]
```diff
@@ -141,7 +141,16 @@ function install_istio_with_helm() {
 # install istio with default IstioControlPlane CR yaml using istioctl.
 function install_istio_with_istioctl() {
   local CR_PATH="${WD}/istioctl_profiles/${CR_FILENAME}"
+  pushd "${ISTIOCTL_PATH}"
   ./istioctl manifest apply -f "${CR_PATH}" --set "${SET_OVERLAY}" "${EXTRA_ARGS}"
+  popd
+}
+
+function install_telemetryv2_stats() {
+  export SET_OVERLAY="values.telemetry.enabled=true,values.telemetry.v2.enabled=true"
```
Comment (carolynhu) [inline perf/istio-install/setup_istio.sh]: @mandarjog Does this command will enable both metadata exchange and stats filter? If so, how can we separately test the affect of metadata exchange?

Comment (richardwxn) [reply perf/istio-install/setup_istio.sh]: this is equivalent to existing setup(with metadata exchange and stats filter), I will add stackdriver related setup in another PR

File: perf/istio-install/istioctl_profiles/istio-non-sds.yaml [hunk 2]
```diff
@@ -1,10 +1,9 @@
-apiVersion: install.istio.io/v1alpha2
-kind: IstioControlPlane
+apiVersion: operator.istio.io/v1alpha1
```
Comment (mandarjog) [inline perf/istio-install/istioctl_profiles/istio-non-sds.yaml]: Will these changes continue to work with release 1.4?

Comment (richardwxn) [reply perf/istio-install/istioctl_profiles/istio-non-sds.yaml]: this is for master. 1.4 is still using the old API

**Comment by richardwxn** [comment]
@howardjohn @mandarjog Can you please review it?

File: perf/benchmark/run_benchmark_job.sh [hunk 3]
```diff
@@ -115,9 +115,15 @@ function setup_fortio_and_prometheus() {
     export FORTIO_SERVER_POD
 }
 
+#TODO: add stackdriver filter
 function prerun_v2_nullvm() {
-  kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/istio/"${GIT_BRANCH}"/tests/integration/telemetry/stats/prometheus/testdata/metadata_exchange_filter.yaml
-  kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/istio/"${GIT_BRANCH}"/tests/integration/telemetry/stats/prometheus/testdata/stats_filter.yaml
+  export SET_OVERLAY="values.telemetry.enabled=true,values.telemetry.v2.enabled=true"
+  export CR_FILENAME="default.yaml"
+  export EXTRA_ARGS="--force=true"
```
Comment (howardjohn) [inline perf/benchmark/run_benchmark_job.sh]: why?

File: perf/istio-install/istioctl_profiles/default.yaml [hunk 4]
```diff
@@ -1,115 +1,81 @@
...
-              memory: 12G
-            requests:
-              cpu: 4800m
-              memory: 2G
-    enabled: true
+  components:
+    sidecarInjector:
+      enabled: true
+    galley:
+      enabled: true
```
Comment (howardjohn) [inline perf/istio-install/istioctl_profiles/default.yaml]: why set these explicitly? we are going to turn them off

File: perf/istio-install/istioctl_profiles/istio-non-sds.yaml [hunk 5]
```diff
@@ -1,10 +1,9 @@
...
+kind: IstioOperator
 spec:
-  security:
-    components:
-      nodeAgent:
-        enabled: false
+  components:
+    nodeAgent:
+      enabled: false
```
Comment (howardjohn) [inline perf/istio-install/istioctl_profiles/istio-non-sds.yaml]: is the other one even turning on sds? afaik istioctl does not work with node agent

**Pull request merged**